carla/recourse_methods/processing/immutables.py

```diff
@@ -0,0 +1,19 @@
+def encoded_immutables(imtbls, input_order):
```

Comment:
We could use this method if we have a mixture of continuous and categorical features that we want to encode, like in the immutable case. If we want to encode the whole categorical features, it would be easier to call `OneHotEncoder.get_feature_names(...)`. At the moment it is a special case only for immutables.